native/core/src/execution/datafusion/planner.rs

```diff
@@ -1599,6 +1602,24 @@ impl PhysicalPlanner {
         }
     }
 
+    /// Find DataFusion's built-in window function by name.
+    fn find_df_window_func(name: &str) -> Option<WindowFunctionDefinition> {
```

Comment:
Is it possible to reuse the original `find_df_window_func`?
E.g.
```
if let Some(f) = find_df_window_func(name) {
             Some(f)
} else {
              match name.as_str() {
                 "count" => Some(WindowFunctionDefinition::AggregateUDF(count_udaf())),
                 _ => None,
             }
}
```

Reply:
Sounds good. Changed.